lib/tasks/plugins.rake

```diff
@@ -1,5 +1,18 @@
-desc "List Installed plugins"
-task :plugins => :environment do
-  puts 'Collecting plugin information'
-  Foreman::Plugin.all.map{ |p| puts p.to_s }
+namespace :plugins do
+  desc "List Installed plugins"
+  task :list => :environment do
+    puts 'Collecting plugin information'
+    Foreman::Plugin.all.map{ |p| puts p.to_s }
+  end
```

Comment:
This effectively moves the `plugins` to `plugins:list`. I hope this does not cause any issues in packages or something.